pkg/app/piped/cmd/piped/piped.go

```diff
@@ -100,6 +104,7 @@ func NewCommand() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&p.configFile, "config-file", p.configFile, "The path to the configuration file.")
+	cmd.Flags().StringVar(&p.configInGCPSecretManager, "config-in-gcp-secret-manager", p.configInGCPSecretManager, "The resource ID of secret that contains Piped config and be stored in GCP SecretManager.")
```

Comment:
A bit odd. That being said, I don't come up with a better name 😅

Reply:
Yeah, still no good name for this. Just changed it to `config-gcp-secret`.